consensus/power_table_view.go

```diff
@@ -63,7 +63,7 @@ func (v *MarketView) Total(ctx context.Context, st state.Tree, bstore blockstore
 // should probably be increased for v1.
 func (v *MarketView) Miner(ctx context.Context, st state.Tree, bstore blockstore.Blockstore, mAddr address.Address) (uint64, error) {
 	vms := vm.NewStorageMap(bstore)
-	rets, ec, err := CallQueryMethod(ctx, st, vms, mAddr, "getPower", []byte{}, address.Address{}, nil)
+	rets, ec, err := CallQueryMethod(ctx, st, vms, mAddr, "getPower", []byte{}, address.Undef, nil)
```

Comment:
Methods like this are a reason I think we should allow decoding a encoding and decoding a zero length address -- since we are using one here. Maybe I am misunderstanding the usage here?

Reply:
the address used here shouldnt ever actually get serialized.

Reply:
The message created by this call is serialized which causes the addresses to be serialized along with it.